crates/sui-graphql-rpc/src/types/open_move_type.rs

```diff
@@ -12,18 +12,33 @@ pub(crate) struct OpenMoveType {
     signature: OpenMoveTypeSignature,
 }
 
+/// The basic four abilities for a move object
 #[derive(Enum, Copy, Clone, Eq, PartialEq)]
 pub(crate) enum MoveAbility {
+    /// Allows values of types with this ability to be copied.
     Copy,
+    /// Allows values of types with this ability to be popped/dropped.
     Drop,
+    /// Allows the type to serve as a key for global storage operations.
     Key,
+    /// Allows values of types with this ability to exist inside a struct in global storage.
     Store,
 }
 
+/// Module functions, by default, can only be called within the same module.
+/// These internal (sometimes called private) functions cannot be called from
+/// other modules or from scripts.
```

Comment:
I understand this content was borrowed from the Move book, but:

- Just "Functions", "Module functions" seems redundant?  Alternatively, as we are about to make it so that structs can also be public/private, we can use a more generic term than "function", like "Module member'?
- The concept of a "default" visibility seems strange to me. It maybe applies if you are writing the Move code, but if you are looking at a defined function, it always has a visibility, it's never left implicit so I don't think the concept of a default applies.
- We don't have scripts.

How about this:

> The visibility modifier describes which functions can access this module member.